governance.md

```diff
@@ -293,42 +294,27 @@ building proprietary products and services that leverage Project Software and
 Services. Funding acquired by Institutional Partners to work on The Project is
 called Institutional Funding. However, no funding obtained by an Institutional
 Partner can override The Project BDFL and Steering Council. If a Partner has
-funding to do Jupyter/IPython work and the Council decides to not pursue that
+funding to do Matplotlib work and the Council decides to not pursue that
 work as a project, the Partner is free to pursue it on their own. However in
 this situation, that part of the Partner’s work will not be under the
-Jupyter/IPython umbrella and cannot use the Project trademarks in a way that
+Matplotlib umbrella and cannot use the Project trademarks in a way that
 suggests a formal relationship.
 
-To acknowledge institutional contributions, there are two level of Institutional
-Partners, with associated benefits:
+To acknowledge institutional contributions, Institutional Partners
+receive the following benefits:
 
-**Tier 1** = an institution with at least one Institutional Council Member
-
--   Acknowledged on the Jupyter/IPython websites, in talks and T-shirts.
--   Ability to acknowledge their own funding sources on the Jupyter/IPython
+-   Acknowledged on the Matplotlib websites, in talks and T-shirts.
```

Comment:
Do we need to hedge these statements a bit? For example, a few months ago, I gave a talk where I used matplotlib's development process as an example of how OSS works to people who were looking to possibly adopt a community-driven development model. With this, would I have been required to include these institutions in my talk?

Another example... lightning talks? Tutorials? Also, talks by whom? Would there be a difference if I gave a tutorial (as a Core Developer) versus a Community Member?